ingest/genomes/parse_genome_annotations.py

```diff
@@ -0,0 +1,250 @@
+"""Download genome annotations, transform for visualizations, upload to GCS
+
+TODO (SCP-2491): Add automated tests and refactor code moved from single_cell_portal
+"""
+
+import argparse
+import json
+import os
+import subprocess
+import urllib.request as request
+
+# flake8: noqa F403
+
+from .persist_annotation_metadata import *
+from .utils import *
+
+
+def get_ensembl_metadata():
+    """Get organism, assembly, and annotation release metadata from Ensembl
+    """
+    ensembl_metadata = {}
+
+    # API docs: https://rest.ensembl.org/documentation/info/species
+    url = 'https://rest.ensembl.org/info/species?content-type=application/json'
+    with request.urlopen(url) as response:
+        data = response.read().decode('utf-8')
+    ensembl_species = json.loads(data)['species']
+
+    for species in ensembl_species:
+        taxid = species['taxon_id']
+        name = species['name']
+        assembly = species['assembly']
+        strain = species['strain']
+
+        if taxid == '10090' and strain != 'reference (CL57BL6)':
+            # Mouse has multiple annotated assemblies; only use reference assembly
+            continue
+
+        ensembl_metadata[taxid] = {
+            'organism': name,
+            'taxid': taxid,
+            'assembly_name': assembly,
+            'assembly_accession': species['accession'],
+            'release': str(species['release']),
+        }
+
+    return ensembl_metadata
+
+
+def get_ensembl_gtf_urls(ensembl_metadata, scp_species, output_dir):
+    """Construct the URL of an Ensembl genome annotation GTF file.
+
+    Example URL:
+    http://ftp.ensembl.org/pub/release-94/gtf/homo_sapiens/Homo_sapiens.GRCh38.94.gtf.gz
+    """
+
+    gtf_urls = []
+    for species in scp_species:
+        taxid = species[2]
+        organism_metadata = ensembl_metadata[taxid]
+        release = organism_metadata['release']
+        organism = organism_metadata['organism']
+        organism_upper = organism[0].upper() + organism[1:]
+        assembly = organism_metadata['assembly_name']
+
+        origin = 'http://ftp.ensembl.org'
+        dir = '/pub/release-' + release + '/gtf/' + organism + '/'
+        filename = organism_upper + '.' + assembly + '.' + release + '.gtf.gz'
+
+        gtf_url = origin + dir + filename
+        gtf_urls.append(gtf_url)
+        ensembl_metadata[taxid]['gtf_path'] = output_dir + filename
+
+    return [gtf_urls, ensembl_metadata]
+
+
+def transform_ensembl_gtf(gtf_path, ref_dir):
+    """Produce sorted GTF and GTF index from Ensembl GTF; needed for igv.js
+    """
+    # Example:
+    # $ sort -k1,1 -k4,4n gencode.vM17.annotation.gtf > gencode.vM17.annotation.possorted.gtf
+    # $ bgzip gencode.vM17.annotation.possorted.gtf
```

Comment:
I have the same comment here that I had in https://github.com/broadinstitute/single_cell_portal/pull/99/files#r434173376.  You mentioned a README update in this repo to address it?

Reply:
Done in 11ecf0a.  I had scoped that update among other pre-existing tech debt in SCP-2491, but this was a trivial change.